### PR TITLE
fix: settle bills on correct cash point

### PR DESCRIPTION
**Pull request by amosmachora**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This PR adds a way to settle a bill at the correct cash point. Previously it was assumed that where you create the bill is where it's settled.

### An example of settling a bill at a 'non-default' payment point.
image

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/invoice/payments/utils.ts [hunk 1]
```diff
@@ -51,7 +52,7 @@ export const createPaymentPayload = (
     updatedLineItems.filter((item) => item.paymentStatus === 'PENDING').length === 0 ? 'PAID' : 'PENDING';
 
   const processedPayment = {
-    cashPoint: bill.cashPointUuid,
+    cashPoint: timesheet.cashPoint.uuid,
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/invoice/payments/utils.ts:55]: is it safe to just check for NULLs?

**Comment by ojwanganto** [comment]
@makombe @Omoshlawi @donaldkibet - please help with reviews

**Review by donaldkibet — APPROVED** [review]
Thanks @amosmachora LGTM if we could address @ojwanganto feedback its good to go

**Comment by amosmachora** [comment]
@donaldkibet there was a conflict on this branch I do not know if you addressed it before the merge.

**Pull request merged**